index.js

```diff
@@ -10,9 +10,52 @@ function ask(questionText) {
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  let lowerBound = 0;
+  let higherBound = 100;
+  let guess = average(higherBound, lowerBound);
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  let yesOrNo = await ask("Is it " + guess + "? ");
+  let yesOrNoFinal = yesOrNo.toLowerCase();
+
+  while (yesOrNoFinal !== "yes") {
+    console.log("You entered: " + yesOrNo);
+    let lowOrHigh = await ask("Is it lower or higher? ");
+    let lowOrHighFinal = lowOrHigh.toLowerCase();
+    console.log("You entered: " + lowOrHigh);
+    if (yesOrNoFinal === "yes") {
+      console.log("winner, winner, chicken dinner!");
+      process.exit();
+    }
+    if (yesOrNoFinal !== "yes" && lowOrHighFinal === "higher") {
+      lowerBound = defineLow(lowerBound, guess);
+      guess = average(higherBound, lowerBound);
+    } else if (yesOrNoFinal !== "yes" && lowOrHighFinal === "lower") {
+      higherBound = defineHigh(higherBound, guess);
+      guess = average(higherBound, lowerBound);
+    } else {
+      console.log("What's wrong with you human!? Try again!");
+    }
+    yesOrNo = await ask("Is it " + guess + "? ");
+    yesOrNoFinal = yesOrNo.toLowerCase();
+    if (yesOrNoFinal !== "yes" && yesOrNoFinal !== "no") {
+      console.log("What's wrong with you human!? Try again!");
+    }
+  }
+  console.log("winner, winner, chicken dinner!");
   process.exit();
 }
+
+function defineLow(lowerBound, guess) {
+  return guess - 1;
+}
+
+function defineHigh(higherBound, guess) {
+  return guess + 1;
+}
+
+function average(higherBound, lowerBound) {
+  let number = (higherBound + lowerBound) / 2;
+  return Math.floor(number);
+}
```

Comment:
Although you completed the opposite guess the number game, while also implementing the main guessing game, there is still some work to be done towards an anti-cheat system and a user-defined maximum bound.

more information can be found here: https://bootcamp.burlingtoncodeacademy.com/projects/guess#backlog